utils/utils.jsx

```diff
@@ -2006,3 +2006,23 @@ export function makeIsEligibleForClick(selector = '') {
         return true;
     };
 }
+
+export function numberToFixedDynamic(num, places) {
```

Comment:
Could we add a documenting comment here on what this function does and the params behavior ? I noticed your tests below explain it but some context here would be good